src/test/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/mapper/induction/PreviousQualificationsEntityMapperTest.kt

```diff
@@ -53,7 +53,6 @@ class PreviousQualificationsEntityMapperTest {
     // Then
     assertThat(actual)
       .doesNotHaveJpaManagedFieldsPopulated()
-      .hasAReference()
```

Comment:
Unrelated change; but since the changes the other day to PrisonerQualifications (where we got rid of the displayName fields, but also refactored the entity etc to make fields non-nullable), this assertion is no longer valid because it's not possible for the entity to not have a reference. I've removed the method from the assertion class below

Reply:
this was on my list to sort too